dataplane/standalone/port.cc

```diff
@@ -135,3 +175,5 @@ std::unordered_map<std::string, std::vector<int>> Port::parseLaneMap() {
 
 std::unordered_map<std::string, std::vector<int>> Port::laneMap =
     Port::parseLaneMap();
+
+int Port::nextIdx = 2;
```

Comment:
I think it's clearer if this is part of the header file.

Reply:
if so it causes compile error about redefining this field